testing/regress/ecl/persist_refresh.ecl

```diff
@@ -0,0 +1,67 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+import Std;
+import Std.File AS FileServices;
+
+//
+//version persistRefresh=true
```

Comment:
Yes, the spray changes the input file, whilst the ECL (for the persist remains unchanged).
Using an inline dataset and toggling between 2 varieties would change the ECL crc.

So changing to use one of 2 inline datasets would test an ECL change vs a input file change, but the refresh check in checkPersistUptoDate() is before either crc's are compared with old crc's anyway.
It could test both paths, change a file and change ECL, but I think that's overkill given the condition in checkPersistUptoDate()

@ghalliday - am I missing something?